components/sendgrid/sources/delivery-events/delivery-events.js

```diff
@@ -0,0 +1,84 @@
+const common = require("../common/http-based");
+
+module.exports = {
+  ...common,
+  key: "sendgrid-delivery-events",
+  name: "Delivery Events (Instant)",
+  description: "Emit an event when a delivery event is detected in SendGrid",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    ...common.props,
+    processed: {
+      type: "boolean",
+      label: "Processed",
+      description: `
```

Comment:
hm i suppose these descriptions with newlines are probably okay for when embedded in html but maybe we shouldn't do them like this? (i suppose we could strip out in other clients maybe.. hm..)

Reply:
That is actually I question I raised in another PR. If descriptions are a free-form text, and can be arbitrarily long, then at some point someone will add a very long line to the code.
🤷🏻‍♂️